controller/node_controller.go

```diff
@@ -1356,7 +1360,30 @@ func (nc *NodeController) getNewAndMissingOrphanedReplicaDataStores(diskName, di
 	return newOrphanedReplicaDataStores, missingOrphanedReplicaDataStores
 }
 
-func (nc *NodeController) deleteOrphans(node *longhorn.Node, diskName string, diskInfo *monitor.CollectedDiskInfo, missingOrphanedReplicaDataStores map[string]string) error {
+func (nc *NodeController) deleteOrphansForEngineAndReplicaInstances(node *longhorn.Node) error {
+	nc.logger.Debugf("Deleting orphaned instances to evict node %v", node.Name)
```

Comment:
The function actually deletes orphan CRs.

```suggestion
	nc.logger.Infof("Deleting orphans on evicted node %v", node.Name)
```